### PR TITLE
PEW's path addresses / divider in addition to os.sep divider using co…

### PR DESCRIPTION
**Pull request by grace-harper-ibm**
…mponent file paths

### CODE REVIEW

File: qiskit_metal/_gui/widgets/library_new_qcomponent/parameter_entry_window.py [hunk 1]
```diff
@@ -11,52 +11,77 @@
...
+# pylint: disable-msg=inconsistent-return-statements
+# pylint: disable-msg=attribute-defined-outside-init
+# pylint: disable-msg=broad-except
+# pylint: disable-msg=import-outside-toplevel
+# pylint: disable-msg=relative-beyond-top-level
+# pylint: disable-msg=import-error
+# pylint: disable-msg=too-few-public-methods
+# pylint: disable-msg=too-many-instance-attributes
+# pylint: disable-msg=invalid-name
+# pylint: disable-msg=no-name-in-module
```
Comment (marcolincs) [inline qiskit_metal/_gui/widgets/library_new_qcomponent/parameter_entry_window.py:26]: Oh wow, should we maybe disable some of these globally, rather that at file level? This seem to be quite the lengthy list.
Note deleting those altogether could be an option that does not hinder CI/CD. Yapf only complained about about line 459 (previous 420).

I am good either way, we can keep them now and try to remember to remove them later before running the global linting cleanup.

**Review by marcolincs — APPROVED** [review]
That is quite the cleanup. Thanks
Maybe let's not squash this one(?)
Left a note, no need for me to re-review regardless of decision.
I did not functionally test this but eyeballing it I see nothing off. Maybe @grace-harper-ibm can give it one last functional test, since she had the test-case for her correction?

File: qiskit_metal/_gui/widgets/library_new_qcomponent/parameter_entry_window.py (hunk 1 above)
Comment (JeremyDrysdale-ibm) [reply qiskit_metal/_gui/widgets/library_new_qcomponent/parameter_entry_window.py:26]: Quite possibly, but that's out of scope for this PR.

**Review by JeremyDrysdale-ibm — APPROVED** [review]
LGTM

**Pull request merged**